docker-app/qfieldcloud/core/serializers.py

```diff
@@ -180,6 +182,10 @@ def get_members(self, obj):
             ).values("member__username")
         ]
 
+    def get_teams(self, obj):
+        teams = Team.objects.filter(team_organization=obj)
+        return TeamSerializer(teams, many=True).data
```

Comment:
```suggestion
    def get_teams(self, obj: Organization) -> list[str]:
        """Implementation of `SerializerMethodField` for `teams`. Returns list of team names."""
        return [
            t.team_name
            for t in Team.objects.filter(
                team_organization=obj
            ).values("username")
        ]
```

Just print the team names, not all the info. Similar to what we do with the organization members.

Reply:
Done.. 

Regarding docstring, do you prefer to do it this way? 
  
    **Retrieve and serialize teams associated with a given organization.

    Args:
        obj (Organization): The organization object for which blah blah...

    Returns:
        blah blah blah**

Reply:
Ideally it should be like this, yes. All args and return value should be described.

Sometimes for smaller functions it is acceptable to have single line docstring. This example is a good one -> the function is very short, we have single parameter and clear return value, and the function is a callback that is very well known within DRF. Even if you add the `Args` and `Returns` there is not much added value.

Reply:
Fair enough!